blog/notatki-graficzne/index.md

```diff
@@ -0,0 +1,149 @@
+---
+title: 'Odręczne notatki graficzne w warsztacie technical writera'
+authors: niewiarowski
+date: '2024-12-19'
+tags:
+  - 'dobre-praktyki'
+  - 'warsztat'
+  - 'notatki'
+  - 'facylitacja-graficzna'
+  - 'grafika'
+coverImage: 'upload_0085.png'
+---
+
+Czy analogowe notatki mają prawo bytu w dzisiejszym cyfrowym świecie? Uważam, że
+tak, a przynajmniej "u mnie to działa" :). Zapraszam do zapoznania się z metodą
+tworzenia graficznych notatek.
+
+<!--truncate-->
+
+## Szkolenie z tworzenia graficznych notatek
+
+![](images/upload_0086.png)
+
+W marcu 2023 wziąłem udział w szkoleniu prowadzonym przez
+[Gabrielę Borowczyk](https://www.linkedin.com/in/gabriela-borowczyk-%E2%9C%8F-55558126/).
+Otrzymałem solidną porcję wiedzy o technikach tworzenia różnego rodzaju napisów,
+elementów dekoracyjnych, liter i rysunków. Są to umiejętności pomocne zwłaszcza
+dla scrum masterów podsumowujących wizualnie spotkania, warsztaty i spontaniczne
+burze mózgów. Szkolenie, w którym wziąłem udział z innymi pracownikami Hitachi
+Energy, było bardzo sympatyczne i intensywne. Mieliśmy okazję podkręcić
+kreatywność na wyższe obroty. Ubawiłem się przednio, szczególnie, że ostatni raz
+z takim zapałem rysowałem w liceum, a później było coraz mniej do tego okazji.
+Sam temat był dla mnie ważny o tyle, że moją odwieczną bolączką był charakter
+pisma zbliżony do szyfru lub recepty wypisanej przez lekarza. Było to krzepiące,
+że mimo mojego awangardowego stylu pisania, również i ja byłem w stanie się
+nauczyć jak pisać na plakatach dostatecznie wyraźnie. Dowiedziałem się też o
+graficznych facylitatorach prowadzących warsztaty z użyciem własnoręcznie
+narysowanych materiałów, a także o ilustratorach spotkań i wydarzeń, tworzących
+wizualne podsumowania, które są świetnymi materiałami edukacyjnymi.
+
+## I co dalej zrobić z tą wiedzą?
+
+![](images/upload_0087.png) Warsztaty ukończyłem, dyplom otrzymałem. Przez
+dłuższy czas nie tykałem flamastra, aż pewnego pięknego dnia stwierdziłem, że
+wypróbuję te umiejętności w codziennej praktyce speca od dokumentacji i autora
+radosnej twórczości po godzinach. Poszerzyłem mój zestaw pisaków, zakupiłem
+[biuwar (moje nowe ulubione słowo ;)](https://sjp.pwn.pl/slowniki/biuwar.html) i
```

Comment:
Na tej stronie co jest podlinkowana nie ma wyjaśnienia tego terminu. Może lepiej podlinkować tę stronę: https://kalendarzecsk.pl/blog/52,biuwary---co-to-jest-i-jak-mozna-ich-uzywac

Reply:
zly link. chodzilo mi o : https://sjp.pwn.pl/poradnia/haslo/Zagadkowy-biwuar;17365.html - jest i bogata historia i definicja. podmieniam na pwn:)

Reply:
Ale zauważ, że tu jest "biwuar" a nie "biuwar". Czy "biuwar" to przekręcona nazwa "biwuaru" czy alternatywna nazwa? 🤔